### ISSUE TITLE
PageViews not working with GA

### ISSUE THREAD

**Issue by ganySA** [angulartics/angulartics2#342]
Hi

I am using Angular 8 RC. I have installed and configured as per the instructions but i do not see any events or pageviews coming through to GA. I see only '/' as the page. 

I must admit i dont fully understand the instructions. I am using @angular/router the default. What should i be using Angulartics2RouterlessModule? 

```
    Angulartics2Module.forRoot({
      pageTracking: {
        clearHash: true,
        clearQueryParams: true,
      }
    }),
    Angulartics2RouterlessModule.forRoot(),
```

**Comment by irux** [comment]
Yes, the documentation is really weird sometimes. It could be really useful if someone can help with it. 
@ganySA did you solve it ? I am having only / on GA too, but I am using Google tag manager and Angular7

**Comment by JohannesHoppe** [comment]
Any progress on this?! I realised after a long timer of tracking to few data, that the library must be broken here. 😞 

> Command ignored. Unknown target: undefined

As far as I understand, the `create` event is not happening at all!  
<img width="668" alt="Screen Shot 2019-12-10 at 13 37 39" src="https://user-images.githubusercontent.com/640639/70532453-5deae000-1b57-11ea-8a35-bddb1667a584.png">

**Comment by JohannesHoppe** [comment]
Please ignore my last post. It was an error on my side.
